### PR TITLE
create signUp user lawyer

### PR DESCRIPTION
**Pull request by TroJanBoi**

### CODE REVIEW

**Comment by TroJanBoi** [comment]
create signUp user lawyer

File: Frontend/legal-app/app/login/page.tsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        <div className="flex justify-center items-center bg-white h-screen w-screen font-kanit">
+            <div className="relative flex flex-col justify-around items-center text-center bg-[url(https://www.figma.com/file/LlvSLLtbFgjKjRn7ieW5Pu/image/16f11896ac0171df49a84ace75fd953eba2b6ed9)] bg-center bg-cover w-[22%] h-screen p-8">
+                <div className="">
+                    <h1 className="text-4xl text-white">Login</h1>
+                </div>
+                <div className="">
+                    <h1 className="text-xl text-white">Log in to access expert legal assistance at any time.</h1>
+                </div>
+                <div className="">
+                    {/* <h1 className="text-4xl text-white">Login</h1> */}
```
Comment (Nxwbtk) [inline Frontend/legal-app/app/login/page.tsx]: del comment

File: Frontend/legal-app/app/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+const LoginPage = () => {
+
+    const { register, handleSubmit, watch, formState: { errors }} = useForm<Inputs>()
+    const onSubmit: SubmitHandler<Inputs> = (data) => console.log(data);
+
+    return (
+        <div className="flex justify-center items-center bg-white h-screen w-screen font-kanit">
+            <div className="relative flex flex-col justify-around items-center text-center bg-[url(https://www.figma.com/file/LlvSLLtbFgjKjRn7ieW5Pu/image/16f11896ac0171df49a84ace75fd953eba2b6ed9)] bg-center bg-cover w-[22%] h-screen p-8">
```
Comment (Nxwbtk) [inline Frontend/legal-app/app/login/page.tsx]: Why figma url?

File: Frontend/legal-app/app/login/page.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                        Don't have an account ? <Link href="/signUp" className="text-mainColor border-b-2 border-mainColor">Sign Up Here!</Link>
+                    </div>
+                </div>
+            </div>
+            <div className="flex justify-center w-[15%] h-full">
+                <h1 className="text-4xl mt-[15%]">Logo</h1>
+            </div>
+        </div>
+    );
+}
```
Comment (Nxwbtk) [inline Frontend/legal-app/app/login/page.tsx:63]: format document

File: Frontend/legal-app/app/signUp/page.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+import { useRouter } from "next/navigation";
+
+const signUpPage: React.FC = () => {
+
+    const   router = useRouter();
+
+    return (
+        <div className="flex justify-center items-center bg-white w-screen h-screen font-kanit">
+            <div className="flex justify-center items-center w-full h-screen">
+                <img src="https://www.figma.com/file/LlvSLLtbFgjKjRn7ieW5Pu/image/16f11896ac0171df49a84ace75fd953eba2b6ed9" className="object-cover w-full h-full" alt="" />
```
Comment (Nxwbtk) [inline Frontend/legal-app/app/signUp/page.tsx]: Save image to public folder

File: Frontend/legal-app/app/signUp/page.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                        className="w-full text-4xl bg-mainColor hover:bg-subColor text-white p-8"> ทนาย </Button>
+                        <Button
+                         onClick={() => router.push('/signUp/user')}
+                         className="w-full text-4xl bg-mainColor hover:bg-subColor text-white p-8  mt-3"> ผู้ใช้งานทั่วไป </Button>
+                    </CardContent>
+                </Card>
+            </div>
+        </div>
+    )
+};
```
Comment (Nxwbtk) [inline Frontend/legal-app/app/signUp/page.tsx:38]: Format document

**Review by Nxwbtk — APPROVED** [review]
Nice

**Pull request merged**